docs/configdoc.md

```diff
@@ -0,0 +1,117 @@
+# L3AFD Config Options Documentation
+
+See [l3afd.cfg](https://github.com/l3af-project/l3afd/blob/main/config/l3afd.cfg) for a full example configuration.
+
+
+```
+[DEFAULT]
+
+[l3afd]
+pid-file: ./l3afd.pid
+datacenter: dummy
+bpf-dir: /dev/shm
+bpf-log-dir:
+kernel-major-version: 4
+kernel-minor-version: 15
+shutdown-timeout: 1s
+http-client-timeout: 10s
+max-nf-restart-count: 3
+bpf-chaining-enabled: true
+swagger-api-enabled: false
+# PROD | DEV
+environment: PROD
+....
+```
+
+### Below is the detailed documentation for each field
+
+
+## [l3afd]
+| FieldName     | Example       | Description     |
+| ------------- | ------------- | --------------- |
+|pid-file| `"l3afd.pid"`  |It contains process id of eBPF Programs |
```

Comment:
I'm guessing you mean
```suggestion
|pid-file| `"l3afd.pid"`  |The name of a file that contains process id of eBPF Programs |
```
But is a path legal or only the filename?   Where is the format of this file documented?
Is the file something that l3afd writes to?  (I assume so, but say so)